### DISCUSSION TITLE
DE-CIX Munich recognized as HE

### DISCUSSION THREAD

**Discussion by tomasconstant** [peering-manager/peering-manager#624] [Q&A] [answered]
<img width="1157" alt="Screen Shot 2022-09-23 at 3 00 29 PM" src="https://user-images.githubusercontent.com/95768609/192039322-5510ab7d-0962-45a0-9701-0d520d6e09a3.png">
DECIX Munich route servers are recognized as Hurricane Electric peers, see attached picture. I think that peering manager somehow is not recognizing AS47228 (DECIX Munich RS).

**Comment by gmazoyer** [comment, marked as answer]
The screenshot shows the list of sessions that you have created. These can be therefore fixed by changing the AS for the affected sessions (or deleting them and adding them once again from the available peers view). I've checked what PM imports from PeeringDB regarding DE-CIX RS in Munich as well as HE, it looks OK.

If you still the wrong data in the available peers view, I'd suggest to clear PeeringDB cache and re-generate it to have fresh data.
